### PR TITLE
Re-run weight baseline

### PR DESCRIPTION
**Pull request by athei**
Re-run the benchmarks to check if they stay consistent.

https://weights.tasty.limo/compare?repo=substrate&threshold=10&path_pattern=frame%2F**%2Fsrc%2Fweights.rs&method=guess-worst&ignore_errors=true&unit=time&old=as-weights-gcp&new=at/weights

### CODE REVIEW

File: frame/contracts/src/weights.rs [hunk 1]
```diff
@@ -730,10 +730,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `1324`
 		//  Estimated: `0`
-		// Minimum execution time: 340_676 nanoseconds.
-		Weight::from_ref_time(336_566_074)
-			// Standard Error: 8_433
-			.saturating_add(Weight::from_ref_time(12_773_098).saturating_mul(n.into()))
+		// Minimum execution time: 341_790 nanoseconds.
+		Weight::from_ref_time(648_404_995)
```
Comment (agryaznov) [inline frame/contracts/src/weights.rs:734]: wow, that's a significant increase

**Comment by oleg-plakida** [comment]
/cmd queue -c bench $ pallet dev contracts

**Comment by ggwpez** [comment]
Looks like 13 cases went up by over 30% while the other 104 stayed consistent. Maybe we have to increase the `INSTR_BENCHMARK_BATCHES` to get them to be consistent for the VMs.

**Comment by ggwpez** [comment]
> The thing is: It was working before. We can experiment with those. But this shouldn't be done on a live codebase. My work is completely stalled now. So please roll back those changes at least for substrate. Then you can find a working config on a branch. Once everything works we can switch over. But right now I have pressing deadlines and need to merge stuff. And I can't do that while flying blind with benchmarks.

Yea, maybe we can swap the labels back to the old machines @oleg-plakida? Then fix it in https://github.com/paritytech/substrate/pull/13316 and merge after it stays consistent.

**Comment by alvicsam** [comment]
@athei you can run benchmarks on the old runner with command 

```
/cmd queue -c bench -v PIPELINE_SCRIPTS_REF=bm-fallback $ pallet dev <pallet_name>
```

Does this unblocks you?

**Comment by Robbepop** [comment]
>@Robbepop Could it be still the problem with cranelift producing unaligned memory operations? I think we fixed it in `wasmi` by not zeroing a huge stack on every instantiate? But this was only a 10% spread.

I cannot remember to have noticed that the unaligned memory issue has been fixed in Wasmtime.

Not so much has changed in `wasmi` with respect to Wasm instantiation besides a bug fix that should improve performance for imported memories and tables significantly.

**Comment by athei** [comment]
> > @Robbepop Could it be still the problem with cranelift producing unaligned memory operations? I think we fixed it in `wasmi` by not zeroing a huge stack on every instantiate? But this was only a 10% spread.
> 
> I cannot remember to have noticed that the unaligned memory issue has been fixed in Wasmtime.
> 
> Not so much has changed in `wasmi` with respect to Wasm instantiation besides a bug fix that should improve performance for imported memories and tables significantly.

I know it isn't fixed in cranelift. But I think we shrank the initial stack size in wasmi massively to it doesn't have a big impact anymore (or at least make it configurable).

**Comment by ggwpez** [comment]
/bot bench $ pallet dev pallet_contracts

Second run is a bit more consistent over-all with one outlier to `instr_br_table` @ 157%, the other `br` instructions are quite high as well :thinking:  [third-diff-to-second.zip](https://github.com/paritytech/substrate/files/10687743/third-diff-to-second.zip)

**Comment by ggwpez** [comment]
bot clean

File: frame/contracts/src/weights.rs [hunk 2]
```diff
@@ -219,10 +219,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `270 + c * (1 ±0)`
 		//  Estimated: `0`
-		// Minimum execution time: 29_127 nanoseconds.
-		Weight::from_ref_time(15_195_256)
-			// Standard Error: 68
-			.saturating_add(Weight::from_ref_time(57_263).saturating_mul(c.into()))
+		// Minimum execution time: 36_506 nanoseconds.
+		Weight::from_ref_time(40_743_562)
```
Comment (agryaznov) [inline frame/contracts/src/weights.rs:223]: significant increase

File: frame/contracts/src/weights.rs [hunk 3]
```diff
@@ -241,10 +241,10 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `771`
 		//  Estimated: `0`
-		// Minimum execution time: 318_332 nanoseconds.
-		Weight::from_ref_time(375_011_603)
-			// Standard Error: 33
-			.saturating_add(Weight::from_ref_time(35_317).saturating_mul(c.into()))
+		// Minimum execution time: 471_172 nanoseconds.
+		Weight::from_ref_time(499_386_220)
```
Comment (agryaznov) [inline frame/contracts/src/weights.rs:245]: same

File: frame/contracts/src/weights.rs [hunk 4]
```diff
@@ -271,14 +271,14 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `257`
 		//  Estimated: `0`
-		// Minimum execution time: 4_692_895 nanoseconds.
-		Weight::from_ref_time(652_352_191)
-			// Standard Error: 378
-			.saturating_add(Weight::from_ref_time(124_393).saturating_mul(c.into()))
+		// Minimum execution time: 4_501_838 nanoseconds.
+		Weight::from_ref_time(899_786_250)
```
Comment (agryaznov) [inline frame/contracts/src/weights.rs:275]: same

File: frame/contracts/src/weights.rs [hunk 5]
```diff
@@ -302,12 +302,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
...
 		//  Measured:  `533`
 		//  Estimated: `0`
-		// Minimum execution time: 2_397_120 nanoseconds.
-		Weight::from_ref_time(47_153_721)
-			// Standard Error: 12
-			.saturating_add(Weight::from_ref_time(2_268).saturating_mul(i.into()))
-			// Standard Error: 12
-			.saturating_add(Weight::from_ref_time(2_401).saturating_mul(s.into()))
+		// Minimum execution time: 2_359_219 nanoseconds.
+		Weight::from_ref_time(338_145_767)
```
Comment (agryaznov) [inline frame/contracts/src/weights.rs:306]: oh gosh

**Pull request closed without merging**